examples/EventV1/EventV2.json

```diff
@@ -0,0 +1,26 @@
+        "type": "nae.pknca.input.data"
+      },
+      {
+        "id": "ProjectId:5e1f4591-9868-4663-9f32-524e37ccc1d2,ResourceId:cd956b5f-ab40-4fd5-aada-1cf04005b7bd",
+        "datacontenttype": "application/json",
+        "source": "{\"routeOfAdministration\":\"EV\",\"matrix\":\"Plasma\",\"method\":\"lin up/log down\",\"requiredPointsInTerminalPhase\":3,\"partialAreasCount\":3,\"partialAreasStartTimes\":[0,0,0],\"partialAreasStopTimes\":[5.1,6.3,13],\"ncaTimeField\":\"MRRELTM\",\"ncaConcentrationField\":\"ACONC\",\"uniquePkProfilesSortingVars\":[\"SUBJID\",\"TRTP\",\"AVISITN\"],\"finalOutputParams\":[\"Cmax\",\"AUClast\",\"Tlast\",\"Tmax\",\"Clast\"],\"minAdjustedR2\":0.8,\"aucExtrapolationCriterion\":40,\"profiles\":[{\"profileKeys\":{\"SUBJID\":\"foo\",\"TRTP\":\"foo1\",\"AVISITN\":\"foo3\"},\"startTime\":\"NA\",\"stopTime\":\"NA\"}]}",
```

Comment:
This is the second of two largest detours.

1. Instead of pointing to a directory I'm inlining the entire JSON string with the `source` attribute.
2. I removed the `name` attribute.

Reply:
As commented elsewhere, there is no need to serialize the JSON to string here.

Reply:
Please see the [CloudEvents Spec](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md). The source is a reference to the context of the event, the data should go into the "data" field.

Reply:
Moved `source` to `data`. I updated my code in `projectx` and the sample here.